### DISCUSSION TITLE
Zeus bridge connection error

### DISCUSSION THREAD

**Discussion by emircanerkul** [zepp-health/zeppos-forum#137] [Developer Feedback]
Hi, when I try to install miniapp via zeus bridge i got 

`info  Install lite app result: bluetooth channel is not available. Please try again later` error. How can I solve this? I able to build my custom code with V1 for mi 7 but can not install.

BTW I  use Zepp app and mi band 7. All the time I change targets. (I have to stick l66 V1 due to mi7 specs)

Also are they any plan to support mi 7 band officially?

Thank you

```json

"l66": {
      "module": {
        "page": {
          "pages": [
            "page/index"
          ]
        }
      },
      "platforms": [
        {
          "name": "l66",
          "deviceSource": 260
        },
        {
          "name": "l66w",
          "deviceSource": 261
        },
        {
          "name": "l66_1",
          "deviceSource": 262
        },
        {
          "name": "l66w_2",
          "deviceSource": 263
        },
        {
          "name": "l66_3",
          "deviceSource": 264
        },
        {
          "name": "l66_4",
          "deviceSource": 265
        },
        {
          "name": "l66_5",
          "deviceSource": 266
        }
      ],
      "designWidth": 192
    }
```

**Comment by MyoungXUE** [comment]
Thank you for your feedback, I will feedback this issue to the relevant colleagues.

**Reply by emircanerkul** [reply]
Thank you @MyoungXUE

**Reply by Xuemuyang** [reply]
Sorry, mi band 7 is not an officially supported device and cannot be installed via the Zepp App.

**Reply by emircanerkul** [reply]
@Xuemuyang I know but after applying these https://prog.world/we-write-an-application-on-mi-band-7-with-a-service-part/ I able to install watch faces but miniapps. May there any other problem? You also can check for Amazfit Band 7.

To make sure is Amazfit Band 7 supports mini apps? I think yes because the article says seems like that. But I faced bluetooth channel is not available error

**Comment by emircanerkul** [comment]
I used https://codeberg.org/Freeyourgadget/Gadgetbridge
